news.js

```diff
@@ -0,0 +1,39 @@
+class News {
+  constructor() {
+    this.articles = [];
+  }
+
+  fetchAllNews(date, pages) {
+    for (let i = 0; i < pages; i++){
+      fetch(`http://news-summary-api.herokuapp.com/guardian?apiRequestUrl=http://content.guardianapis.com/search?from-date=${date}`)
+      .then(response => response.json())
+      .then(data => {
+        data.response.results.forEach(article => {
+          this.createArticleDiv(article);
+        })
+      })
+    }
+  };
+
```

Comment:
Finds out how many pages to grab then fetches everything! I can't think of any other way to access how many pages to get. perhaps there might be a simpler way without having to do two calls though.